src/app/state/operations/datavault.ts

```diff
@@ -78,3 +78,31 @@ export const swapDataVaultContent = (client: DataVaultWebClient, key: string, co
 export const getStorageInformation = (client: DataVaultWebClient) => (dispatch: Dispatch<any>) =>
   client.getStorageInformation()
     .then((storage: DataVaultStorageState) => dispatch(receiveStorageInformation({ storage })))
+    .catch((err: Error) => console.log('could not get storage', err))
```

Comment:
This hides the error

Reply:
I will remove that line to allow the error to show.